ALevelHomework3/Homework3.2/Program.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+class Program
+{
+    static void Main()
+    {
+        Console.Write("Enter a number with a virgule: ");
+        double number;
+
+        if (double.TryParse(Console.ReadLine(), out number))
+        {
+            if (number == 0)
+            {
+                Console.WriteLine("You entered a zero number.");
+            }
+            else if (number % 1 == 0)
+            {
+                Console.WriteLine("This number doesn't contain a virgule.");
+            }
+            else
+            {
+                switch (Math.Sign(number))
+                {
+                    case 1:
```

Comment:
You can add this check  **_if (number == 0)_** to the switch_case here and inform user that the input value is 0